main.cpp

```diff
@@ -122,6 +125,18 @@ class CDnsSeedOpts {
           break;
         }
 
+        case 's': {
+          if (!strcmp(optarg, "NODE_NETWORK"))
+            requiredNodeFlags = NODE_NETWORK;
```

Comment:
Maybe use `|=` instead of `=`, so that this flag can be specified multiple times?